etc/inc/archiver-common.inc

```diff
@@ -0,0 +1,44 @@
+# This file is overwritten during software install.
+# Persistent customizations should go in a .local file.
+include archiver-common.local
+
+# common profile for archiver/compression tools
+
+blacklist ${RUNUSER}/wayland-*
```

Comment:
I think we can `blacklist ${RUNUSER}`.

Reply:
@rusty-snake Thanks for having a look at this draft. I'm in the middle of testing all my archivers including their .local overrides. Will add `blacklist ${RUNUSER}` to the tests but indeed, I agree this shouldn't pose any problems. The plan is to bring existing profiles in an easier-to-change state using archive-common.inc as baseline and do specific changes in seperate PR's once this is merged. Shouldn't take all that long to get the first part finished.

Reply:
@rusty-snake I added `blacklist ${RUNUSER}` to archiver-common.inc. What's the point in having that in combination with `blacklist ${RUNUSER}/wayland-*` again? I see such a doubling in several other profiles (curl, dig, drill, file, wget to name a few) but I can't say I actually understand the why. If ${RUNUSER} is blacklisted, shouldn't that imply ${RUNUSER}/wayland-* (and any other subdir for that matter) is blacklisted too?

Reply:
`blacklist ${RUNUSER}/wayland-*` is useless with `blacklist ${RUNUSER}` and can be simplified, that's right.

The profiles are written with this logic:

```
If not requires(X11) then
    If 'net none' then
        add 'x11 none'
    Else
        add 'blacklist /tmp/.X11-unix'
    Fi
Fi

If 'x11 none' or 'blacklist /tmp/.X11-unix' then
    add 'blacklist ${RUNUSER}/wayland-*'
Fi

If possible_to_add(wruc) then
    add wruc
Fi

If get_used_commands(wruc) == nil then
    replace wruc with 'blacklist ${RUNUSER}'
Fi
```

Reply:
@rusty-snake Thanks for the explanation! I'll do the needed changes in a minute, before I forget. It sure is a luxury to have you on board :-)